src/MLJTSVDInterface.jl

```diff
@@ -19,15 +19,12 @@ decomposition (SVD). Contrary to PCA, this estimator does not center the data be
 the singular value decomposition. This means it can work with sparse matrices efficiently.
 
 """
-mutable struct TSVDTransformer <: MLJModelInterface.Unsupervised
-    nvals::Int
-    maxiter::Int
-    rng::Union{Int, AbstractRNG}
+MMI.@mlj_model mutable struct TSVDTransformer <: MLJModelInterface.Unsupervised
+    nvals::Int = 2
+    maxiter::Int = 1000
+    rng::Untion{Int, AbstractRNG} = GLOBAL_RNG
```

Comment:
typo: Untion

Reply:
lol thanks

Reply:
looks like that is fixed in the most recent commit